pkg/api/shardprowconfig/shardprowconfig.go

```diff
@@ -108,6 +108,9 @@ func ShardProwConfig(pc *prowconfig.ProwConfig, target afero.Fs, f ShardProwConf
 				return nil, fmt.Errorf("failed to deepcopy tide query %+v: %w", query, err)
 			}
 
+			queryCopy.Orgs = nil
```

Comment:
Why do we need to do this here? This can break other tools.

Reply:
See above https://github.com/openshift/ci-tools/pull/2800#discussion_r867787486
This function was extracted to common tools by me.